### PR TITLE
shh rpc test, initial version

### PR DESCRIPTION
**Pull request by gluk256**

### CODE REVIEW

**Comment by gluk256** [comment]
never mind the errors -- they are not mine

File: test/libwhisper/shhrpc.cpp [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+			{
+				Message msg = web3->whisper()->envelope(i).open(web3->whisper()->fullTopics(w));
+				last = RLP(msg.payload()).toInt<unsigned>();
+				if (received.insert(last).second)
+					result += last;
+			}
+		}
+	});
+
+	for (unsigned i = 0; i < 2000 && (!host2.haveNetwork() || web3->haveNetwork()); i += step)
```
Comment (subtly) [inline test/libwhisper/shhrpc.cpp]: !host2.have || web3->have
Should this be !host2.have || !web3->have?

**Comment by gluk256** [comment]
fixed

**Comment by gluk256** [comment]
never mind the errors

**Comment by gavofyork** [comment]
can we ping this to get the build bots green-lighting it?

**Comment by gluk256** [comment]
Done. Two timeouts and one unrelated error.

**Comment by gluk256** [comment]
@gavofyork : this PR is just new tests. I believe, it would be still worth merging, even if there were problems somewhere else.

**Pull request merged**